src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -285,6 +291,7 @@ private Bucket newCloudBucket(GoogleBucketResource bucketResource) {
             // See here for possible values: http://g.co/cloud/storage/docs/storage-classes
             .setStorageClass(StorageClass.REGIONAL)
             .setLocation(gcsConfiguration.getRegion())
+            .setVersioningEnabled(doVersioning)
```

Comment:
I think it makes sense to turn on versioning but from what I can tell, this doesn't keep users from deleting objects.  Is that ok?